src/main/io/ledstrip.c

```diff
@@ -980,6 +983,11 @@ void pgResetFn_specialColors(specialColorIndexes_t *instance)
     memcpy_fn(instance, &defaultSpecialColors, sizeof(defaultSpecialColors));
 }
 
+void pgResetFn_ledType(ledType_t *instance)
+{
+    defaultLedType.t = RGB;
+    *instance = defaultLedType;
```

Comment:
yes, I wasn't too sure about that part.
I don't understand what you mean exactly with "auto variabel". Could you please elaborate...?